commands/addons/create.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const cli = require('heroku-cli-util')
+const co = require('co')
+
+function parseConfig (args) {
```

Comment:
The CLI already has a pretty advanced flag parser. This is a strange case, and the only one, where it can't parse the options because the CLI isn't aware of what they are. It's definitely not something I want to do anywhere in the CLI because it makes it so there is no way I can generate the help for the options.

So I don't want it in a library, because I don't want us to do this again. It does match the existing logic though.